data/cameras.xml

```diff
@@ -12480,6 +12489,9 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
+	<Camera make="SAMSUNG" model="NX3300" supported="unknown-no-samples">
+		<ID make="Samsung" model="NX3300">Samsung NX3300</ID>
+	</Camera>
```

Comment:
Just verified sample for this one.

Reply:
Could be a straight copy (or even an alias?) of NX3000...